src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -80,6 +88,18 @@ const TablesUserList: React.FunctionComponent<PropsType> = ({
   if (!isRecentVenueUsersLoaded) return <>Loading...</>;
 
   const tables: Table[] = customTables || defaultTables;
+
+  const emptyTables = tables.filter((table) => {
+    if (!table.capacity) return false;
+
+    const usersSeatedAtTable = recentVenueUsers.filter(
+      (u) => u.data?.[venueName].table === table.reference
+    );
+    return table.capacity - usersSeatedAtTable.length > 0;
+  });
+
+  const startTable = emptyTables.length <= ALLOWED_EMPTY_TABLES_NUMBER;
```

Comment:
Our boolean fields should always start with a prefix like `isFoo`, `shouldFoo`, etc

Reply:
fixed in 628728c978f8946518261612288a3324163ca176